### DISCUSSION TITLE
v2.0.0-rc1

### DISCUSSION THREAD

**Discussion by gianni-rosato** [gianni-rosato/svt-av1-psy#28] [Announcements]
*Note: this is a release candidate, not an official release. Expect the lack of polish that usually accompanies pre-release software, as well as this incomplete release changelog. Note that this is also the same changelog as `v1.9.0-rc1` because they are the same version with different numbers.*

## PSY Updates

- Presets got faster, so in addition to Preset -2, we have an even slower Preset -3
- `--sharpness` now accepts negative values
- The SVT-AV1-PSY encoder now supports Dolby Vision encoding via Dolby Vision RPUs. To build with Dolby Vision support, supply pass `--enable-libdovi` to `./build.sh` on macOS/Linux (or `enable-libdovi` to `./build.bat` on Windows).

## Mainline Updates

Encoder
- Improve the tradeoffs for the random access mode across presets:
- Speedup presets MR by ~100% and M0 to M8 by 3-28% while maintaining similar quality levels
- Improved the compression efficiency of presets M9-M13 by 1-4% (!2179)
- Simplified VBR multi-pass to use 2 passes to allow integration with ffmpeg
- Continued adding ARM optimizations for functions with c_only equivalent

Cleanup and bug fixes and documentation
- Various cleanups and functional bug fixes
- Update the documentation to reflect the rate control updates

**Full Changelog**: https://github.com/gianni-rosato/svt-av1-psy/commits/v2.0.0-rc1

<hr /><em>This discussion was created from the release <a href='https://github.com/gianni-rosato/svt-av1-psy/releases/tag/v2.0.0-rc1'>v2.0.0-rc1</a>.</em>

**Comment by libreom** [comment]
What is the breaking change for 2.0

**Reply by gianni-rosato** [reply]
Between this and 1.9, there isn't one. Between this and 1.8, there's a discussion on the mainline repo: https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/2155
